src/attrs-table.cc

```diff
@@ -88,11 +92,13 @@ bool AttributesTable::prepare_entry_buffer(const v8::Local<v8::Object>& pt,
     for (u_int32_t i = 0; i < length; ++i) {
         v8::Local<v8::Value> key = Nan::Get(keys, i).ToLocalChecked();
         v8::Local<v8::String> key_str(key->ToString());
-        if (!ignored_attributes_->IsEmpty() && Nan::HasOwnProperty(Nan::New(*ignored_attributes_), key_str).FromJust()) {
+        v8::String::Utf8Value tag(key);
+        std::string tag_str(*tag);
+
+        if (!ignored_attributes_.empty() && _contains(ignored_attributes_, tag_str)) {
```

Comment:
Oh wait - scratch that. In this new approach you're actually copying the ignored_attributes vector instead of sharing it via a pointer.

So in fact, you should change `ignored_attributes` to be a `std::vector<std::string>*` and then make sure it's non-null before calling `_contains`.